README.md

```diff
@@ -517,6 +517,13 @@ __::isCollection([1, 2, 3]);
 // >> true
 ```
 
+##### [__::isCollection](src/__/objects/isEqual.php)
+Performs a deep comparison between two values to determine if they are equivalent.
+```php
+__::isEqual([1, 1.0]);
```

Comment:
Oops... Accidentally deleted my comment. Are these docs meant for `isCollection()` or `isEqual()`? If it's for `isEqual()`, I think there's one argument missing from the function call.

Reply:
Oh yes, that was bad